### ISSUE TITLE
Question about pressure zones: pressure rise

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1392]
```
Application Version: FDS 553
SVN Revision Number:7031
Compile Date:Fri, 29 Oct 2010
Operating System:Win XP 

Describe details of the issue below:
I have a problem understanding some effect using pressure zones.
In the attached simulations I pressurized a staircase with 6 rooms attached (7 zones).

In RDA_50.fds one door opens, in RDA_51.fds all doors stay closed.
When the door opens, the pressure in the zone above and below rises, although there
is no direct connection between them.

I tried to describe the effect in the attached PDF-document.

Boris
```

Original issue reported on code.google.com by `Info@F-Sim.de` on 2011-05-26 16:40:55

<hr>
- _Attachment: [description.pdf](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1423/comment-0/description.pdf)_
- _Attachment: [RDA_50.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1423/comment-0/RDA_50.fds)_
- _Attachment: [RDA_51.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1423/comment-0/RDA_51.fds)_
- _Attachment: [RDA_50.smv](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1423/comment-0/RDA_50.smv)_
- _Attachment: [RDA_51.smv](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1423/comment-0/RDA_51.smv)_

**Comment by gforney** [comment]
```
I'll take a look at it.
```

Original issue reported on code.google.com by `mcgratta` on 2011-05-26 18:14:13

**Comment by gforney** [comment]
```
Yes, you are correct. The pressures above and below the room increase slightly. This
is due to the fact that the pressure solver used in FDS allows us to specify the exact
boundary conditions only at the external boundaries of the computational domain (the
XB on your MESH line). Inside the domain, the pressure boundary condition is only approximate.
When the door to the room opens, there is a fairly sudden increase in the room pressure.
Because the boundary conditions are not perfect at the floor and ceiling of this room,
there is also a slight increase in pressure across the solid obstructions. 

Susan Kilian can provide some more details of this phenomena if you are interested.
She is working on ways to improve the pressure solver.
```

Original issue reported on code.google.com by `mcgratta` on 2011-05-27 17:32:22

**Comment by gforney** [comment]
```
Kevin,

thanks for these informations.

Boris
```

Original issue reported on code.google.com by `Info@F-Sim.de` on 2011-05-28 08:00:24

**Comment by gforney** [comment]
```
Kevin, I just tried to understand how the pressure zone information is incorporated
into the pressure system. Because it isn't done by corresponding 'internal' boundary
conditions, it must be a part of the right hand side of the pressure system, isn't
it ?

Am I right that one position in the code where this information is added to the right
hand side is in divg.f90? To be more precise, in the PRESSURE_ZONE_LOOPS of DIVERGENCE_PART<1,2>
where it is incorporated to DP (or DDDT respectively) ?? Are there other positions
which are important to know?

Basically, it would be possibly to fix values/bc's also in the interior of the domain
with ScaRC. Surely, I had to analyze the stability of that.  Formerly, I also used
ScaRC to solve problems with multiple boundaries, e.g. a domain with a hole (with predefined
boundary values) in the middle. But how could we know the correct boundary condition
at the transitions of the different pressure zones ?
```

Original issue reported on code.google.com by `kiliansusan` on 2011-05-30 11:11:06

**Comment by gforney** [comment]
```
Each pressure ZONE has a different dp/dt term on the right hand side of the divergence
expression. However, because the boundary condition at solid internal boundaries is
not exact, there will be small errors where a non-zero normal velocity component pushes
air from one compartment to another. This is all just related to the non-exact velocity
condition at solid walls.
```

Original issue reported on code.google.com by `mcgratta` on 2011-06-02 14:30:35
